src/app/sb-admin/components/pages/sb-user/add-edit-user/add-edit-user.component.ts

```diff
@@ -0,0 +1,117 @@
+import { Component } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Organization } from './organizations';
+import { DynamicDialogRef } from 'primeng/dynamicdialog';
+import { UserService } from 'src/app/sb-admin/service/user.service';
+import { Message, MessageService } from 'primeng/api';
+
+@Component({
+  selector: 'app-add-edit-user',
+  templateUrl: './add-edit-user.component.html',
+  styleUrls: ['./add-edit-user.component.scss']
+})
+
+export class AddEditUserComponent {
+  addEditUserForm!: FormGroup
+  submitted: boolean = false;
+  channel: string = "";
+  Organization: Organization[] = [];
+  emailPhoneRequired: boolean = false;
+  organizations: any[] = [];
+  roles: any[] = [];
+  selectedRole: any[] = [];
+  selectedOrganization!: any;
+  messages!: Message[];
+  constructor(
+    private formBuilder: FormBuilder,
+    public ref: DynamicDialogRef,
+    private userService: UserService,
+    private messageService: MessageService
+  ) { }
+
+  ngOnInit(): void {
+    this.getOrganizations();
+    this.initializeForm();
+    this.submitted = false;
+
+    this.roles = [
+      { name: 'Content Creator', value: 'CONTENT_CREATOR' },
```

Comment:
why these roles are hardcoded here?

Reply:
We don't have a roles API, so here are hardcoded roles

Reply:
this should come from the config file then. instead of updating the code from the components - we can remove it from the config file later